script/js_6/index.js

```diff
@@ -0,0 +1,39 @@
+let shopping = [
+    { product: 'apple', quantity: '1', buyer: '1', price: '10' },
+    { product: 'lime', quantity: '3', buyer: '1', price: '20' },
+    { product: 'orange', quantity: '5', buyer: '0', price: '13' },
+    { product: 'banano', quantity: '4', buyer: '1', price: '18' },
+    { product: 'tomato', quantity: '4', buyer: '1', price: '8' },
+    { product: 'paper', quantity: '6', buyer: '0', price: '10' },
+]
+//1.1
+
+shopping.sort(function (a) {
+    if (a.buyer == 1) {
```

Comment:
You can use ternary here:

shopping.sort((item) => item.buyer === 1 ? 1 : -1;)

👍